src/Orleans/Runtime/RequestContext.cs

```diff
@@ -42,6 +41,11 @@ public static class RequestContext
         internal const string ORLEANS_REQUEST_CONTEXT_KEY = "#ORL_RC";
         internal const string PING_APPLICATION_HEADER = "Ping";
 
+#if NETSTANDARD_TODO
+        private static readonly AsyncLocal<Guid> ActivityId = new AsyncLocal<Guid>();
+        private static readonly AsyncLocal<Dictionary<string, object>> CallContextData = new AsyncLocal<Dictionary<string, object>>();
```

Comment:
Previously <code>Trace.CorrelationManager</code>  was used for storing it after extracting from contextData. It could be stored in <code>CallContextData</code> dictionary as well, saving us from additional variable but on cost of more conditional compilation statements.